components/pipeline-service/development/reduce-watcher-cpu-mem-reqs.yaml

```diff
@@ -0,0 +1,7 @@
+---
+- op: replace
+  path: /spec/template/spec/containers/1/resources/requests/cpu
+  value: "100m"
+- op: replace
+  path: /spec/template/spec/containers/1/resources/requests/memory
+  value: "64Mi"
```

Comment:
the old value per https://github.com/openshift-pipelines/pipeline-service/commit/782bf5ca9d1d4cae40d834a0e16dda477185552b#diff-4e66af9db65684795708a50a86c0addbbff2c8ee0314f24861645838423745e2R28